packages/ckeditor5-dev-build-tools/src/plugins/splitCss.ts

```diff
@@ -0,0 +1,318 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { createFilter } from '@rollup/pluginutils';
+import { parse, type Rule, type Declaration, type Stylesheet } from 'css';
+import type { Plugin, GetModuleInfo, OutputBundle, OutputChunk, NormalizedOutputOptions } from 'rollup';
+import MagicString, { Bundle } from 'magic-string';
+
+import { getBanner } from './utils.js';
+
+/**
+ * Filter files only with `css` extension.
+ */
+const filter = createFilter( [ '**/*.css' ] );
+
+export function splitCss(): Plugin {
+	const styles: Record<string, string> = {};
```

Comment:
Consider renaming this to `files` or `stylesheets` so it's easier to remember what this variable holds.

Reply:
I chosen `stylesheetsFiles`.